org.eclipse.scanning.event/src/org/eclipse/scanning/event/AbstractQueueConnection.java

```diff
@@ -225,44 +229,44 @@ public void cleanQueue(String queueName) throws EventException {
 									continue;
 								}
 							}
-	
+
 							if (qbean.getStatus().isFinal()) {
 								final long submitted = qbean.getSubmissionTime();
 								final long current   = System.currentTimeMillis();
 								if (current-submitted > getMaximumCompleteAge()) {
 									removeIds.add(t.getJMSMessageID());
 								}
 							}
-	
+
 						} catch (Exception ne) {
 							logger.warn("Message "+t.getText()+" is not legal and will be removed.", ne);
 							removeIds.add(t.getJMSMessageID());
 						}
 					}
 				}
-	
+
 				// We fail the non-started jobs now - otherwise we could
 				// actually start them late. TODO check this
 				final List<String> ids = new ArrayList<String>();
 				ids.addAll(failIds.keySet());
 				ids.addAll(removeIds);
-	
+
 				if (ids.size()>0) {
-	
+
 					for (String jMSMessageID : ids) {
 						MessageConsumer consumer = qSes.createConsumer(queue, "JMSMessageID = '"+jMSMessageID+"'");
 						Message m = consumer.receive(Constants.getReceiveFrequency());
 						consumer.close();
 						if (removeIds.contains(jMSMessageID)) continue; // We are done
-	
+
 						if (m!=null && m instanceof TextMessage) {
 							MessageProducer producer = qSes.createProducer(queue);
 							final StatusBean    bean = failIds.get(jMSMessageID);
 							bean.setStatus(Status.FAILED);
 							producer.send(qSes.createTextMessage(service.marshal(bean)));
-	
+
 							logger.warn("Failed job "+bean.getName()+" messageid("+jMSMessageID+")");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use the built-in formatting to construct this argument. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2629)